bootstrap/remote/verify.py

```
@@ -305,6 +311,139 @@ def verify_blob_integrity(self) -> list[Issue]:
+        `history.pb2` resolves to an existing resource-snapshot directory.
+        Missing → error (blocks verify exit)."""
+        issues: list[Issue] = []
+        registry = self.head_store.get_registry()
+
+        from bootstrap.remote.models import ServerHistory
+
+        for channel_name in registry.channels:
+            try:
+                head = self.head_store.get_head(channel_name)
+            except Exception:
+                continue
```

Ruff (0.15.18)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)